logistics/handling-money.md

```diff
@@ -0,0 +1,24 @@
+# Handling Money
+
+RustBridge should be free for attendees.
+It is possible to run an event on a very small budget.
+The two most common things that cost money are renting space and providing food for attendees.
+
+Approach local companies or universities for sponsorship.
+They might give you a venue for free or for a very small amount of money.
+If you can get a venue sponsor you already saved yourself what could be your biggest cost. 
+
+Food for attendees is another big cost. Try to find a sponsor to cover that.
+Remember, sponsors get publicity for their businesses or organizations out of sponsorship – which can mean an awful lot to them! But if all else fails, if you’ve got a space but no lunch, ask your participants to bring their own lunch.
+Additionally, you can also make a list of local restaurants and supermarkets nearby. Look for things in walking distance (5-10 minutes of walking at best).
+Make sure to communicate the plan to attendees before the event, so they can prepare for it.
+
+If you require some money but did not find any sponsors, you should contact the [Rust Community Team](mailto:community@rust-lang.org).
+They might be able to connect you to potential sponsors or get a small budget.
+Get in contact as early as possible.
+
+The Community Team is also a good contact to get some swag such as stickers.
+Keep in mind that swag is a nice-to-have, not a must.
+
+You can run a successfull event without a budget.
```

Comment:
`successful`

Reply:
You should always have more `l`s! (Gonna change it...)